### ISSUE TITLE
[Question] [Possible Feature Request] Does dying to a non-player entity reduce your power?

### ISSUE THREAD

**Issue by 42willow** [ickerio/factions#74]
For example: If I tried to swim in lava and died would that decrease my power?

If dying to a non-player entity reduces your power I think there should be an option to toggle this in the config.
On an SMP for example, some people die a lot to mobs and they lose power, I think it would be better if you only lose power if a player or a players' wolf has killed you.

**Comment by BlueZeeKing** [comment]
No, it does not. The next version will include this as an option though

**Comment by BlueZeeKing** [comment]
#71 will include this
